app/livechat/server/config.js

```diff
@@ -555,4 +545,18 @@ Meteor.startup(function() {
 		i18nLabel: 'How_long_to_wait_to_consider_visitor_abandonment',
 		i18nDescription: 'Time_in_seconds',
 	});
+
+	settings.add('Livechat_call_provider', 'none', {
+		type: 'select',
+		public: true,
+		group: 'Omnichannel',
+		section: 'Livechat Call',
```

Comment:
Same goes here as well, by adding Livechat we're restricting the scope of this setting. Lets call it `Calls` or `Audio and Video calls`

Reply:
Also I didn't find any translations for this section within the i18n folder. Please add those as well